### PR TITLE
Cameron

### PR DESCRIPTION
**Pull request by SnugTop**
Author: @SnugTop

This pull request adds email verification to the process of submitting a netid. 

This PR adds:

- a route at api/bootcamp/verify/route.ts
- a function found at utils/emails.ts named sendVerifactionEmail

Note: There are three new env variables needed for the app to work which are:
GMAIL_USER=
GMAIL_PASS=
NEXT_PUBLIC_BASE_URL=

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,14 +15,18 @@
     "@mui/x-charts": "^7.15.0",
     "@supabase/supabase-js": "^2.45.3",
     "next": "14.2.7",
+    "nodemailer": "^6.9.15",
     "react": "^18",
     "react-dom": "^18",
+    "resend": "^4.0.0",
```
Comment (MatthewDavidJordan) [inline package.json]: @SnugTop 
Since we are no longer using resend and instead using nodemailer let's remove this from the package.json

Comment (SnugTop) [reply package.json]: Got it!

File: package.json [hunk 2]
```diff
@@ -16,8 +16,6 @@
     "@supabase/supabase-js": "^2.45.3",
     "next": "14.2.7",
     "nodemailer": "^6.9.15",
-    "react": "^18",
```
Comment (MatthewDavidJordan) [inline package.json:18]: i think you still need react and react dom. Was just suggesting to remove resend. new to these comments my bad if it wasn't clear
@SnugTop

**Review by MatthewDavidJordan — APPROVED** [review]
Looks good now!

**Pull request merged**